websites/H/havesexwith.me/presence.ts

```diff
@@ -0,0 +1,34 @@
+var presence = new Presence({
+    clientId: "782635291105689631" //The client ID of the Application created at https://discordapp.com/developers/applications
+}),
+
+strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+    //You can use this to get translated strings
+});
+
+presence.on("UpdateData", async () => {
+    /*UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+
+    It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.*/
+
+	var presenceData: PresenceData = {
+        largeImageKey: "woman", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageKey: "doggo", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageText: "£2.50 lifetime emails", //The text which is displayed when hovering over the small image
+        details: "On: " + location.hostname, //The upper section of the presence text
+        state: "Being epic", //The lower section of the presence text
+        startTimestamp: Date.now() //The unix epoch timestamp for when to start counting from
+        // endTimestamp: 1577151472000 //If you want to show Time Left instead of Elapsed, this is the unix epoch timestamp at which the timer ends
+    }; /*Optionally you can set a largeImageKey here and change the rest as variable subproperties, for example presenceSata.type = "blahblah"; type examples: details, state, etc.*/
+	
+    if (presenceData.details == null) {
+        //This will fire if you do not set presence details
+        presence.setTrayTitle(); //Clears the tray title for mac users
+        presence.setActivity(); /*Update the presence with no data, therefore clearing it and making the large image the Discord Application icon, and the text the Discord Application name*/
+    } else {
+        //This will fire if you set presence details
+        presence.setActivity(presenceData); //Update the presence with all the values from the presenceData object
+    }
+});
```

Comment:
no var
remove unneeded comments

and this is not enough that get your presence on our website,
there must be some variation in the statuses.

Reply:
Im adding a random comment every time you change url, also, what does Codacy want? Its asking me to change the variables to constants even though I used the file in https://docs.premid.app/en/dev/presence to start